bioio_base/benchmark.py

```diff
@@ -155,16 +162,14 @@ def benchmark(
                 **tests_from_files,
                 **benchmark_test(
                     prefix=test_definition["prefix"],
-                    test=test_definition["test"],
-                    test_file=test_file,
-                    reader=reader,
+                    test=lambda: test_definition["test"](test_file),
```

Comment:
This lambda makes it so that `benchmark_test` doesn't need to be handed the `test_file`